apiaudio/api_resources/media.py

```diff
@@ -0,0 +1,32 @@
+from apiaudio.helper_classes import UploadableResource
+
+
+class Media(UploadableResource):
+    audio_resource_path = "/file/customaudio/"
+    OBJECT_NAME = "file"
+    resource_path = "/file"
+
+    @classmethod
+    def get_download_url(cls, mediaId):
```

Comment:
why arent you making this a DownloadableResource instead?

Reply:
I guess because of url customisation, but I am not sure if we should do it this way. can you check stripe python sdk to see how they do it